pkg/app/handler.go

```diff
@@ -128,5 +129,11 @@ func getResolverOpts(state, gistRoot string) (ro services.ResolverOpts, err erro
 		}
 		ro.GistRoot = g.BigInt()
 	}
+	if signature != "" {
+		if signature != "EthereumEip712Signature2021" {
```

Comment:
move the name to constant

Reply:
Moved to `string(document.EthereumEip712SignatureProof2021Type)` that already exists with `EthereumEip712Signature2021`